.github/workflows/scripts/spdx-tools-java-wrapper.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# SPDX-FileCopyrightText: Maximilian Huber <maximilian.huber@tngtech.com>
+#
+# SPDX-License-Identifier: CC0-1.0
+
+set -euo pipefail
+
+version="1.1.3"
+url="https://github.com/spdx/tools-java/releases/download/v${version}/tools-java-${version}.zip"
+jar="$HOME/spdx-tools-java/tools-java-${version}-jar-with-dependencies.jar"
+
+bootstrap() {
+    if [[ ! -f "$jar" ]]; then
+        mkdir -p "$(dirname "$jar")"
```

Comment:
Can this be dropped? It does not seem to be used.

Reply:
without that, the move of `$jar` below might fail. But might be possible to write it cleaner